python/adaptivity.py.in

```diff
@@ -333,7 +333,65 @@ def detect_colinearity(mesh, tol = 30):
 
     return bfaces_func
 
-
+def adapt_boundary_regions(original_mesh, adapted_mesh, original_regions):
```

Comment:
@taupalosaurus 
I have changed the interface an I have added a call to pragmatic_get_boundaryTags. I have noted something odd. Say that we have a mesh with a corner and say that the corner tip is removed after coarsening. The corner will then have a flat tip. The tag on the edge of this flat tip seems to be lost after the call to pragmatic (the tag on this flat tip will be the same as the tag on the interior of the domain) (see attached pictures).

Is this: expected? a bug? not expected and definitely not a bug and I am doing something wrong?

pragmatic boundary tags:
![get_boundarytags](https://user-images.githubusercontent.com/15267557/28241450-9764c460-698c-11e7-8464-69addf7e2594.png)

correct boundary tags:
![correct_boundarytags](https://user-images.githubusercontent.com/15267557/28241452-a02b8052-698c-11e7-83ac-059e36917a19.png)